index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="description" content="This document describes the mapping of key sequences on an Enlgish keyboard to Plains Cree using the syllabics writing system.">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@exampledev/new.css@1.1.2/new.min.css">
     <link rel="canonical" href="https://www.altlab.dev/plains-cree-syllabics-key-sequences" />
-    <link rel="stylesheet" href="//fonts.googleapis.com/earlyaccess/notosanscanadianaboriginal.css">
+    <!-- <link rel="stylesheet" href="//fonts.googleapis.com/earlyaccess/notosanscanadianaboriginal.css"> -->
```

Comment:
```suggestion
    <link rel="stylesheet" href="//fonts.googleapis.com/earlyaccess/notosanscanadianaboriginal.css">
```

This loads a fallback font, c.f., https://blog.mothertongues.org/tutorial-syllabics-fonts/

Reply:
Oops I meant to uncomment that! I was just testing something.